Shared/Sources/AutoResizingMaskViewSource.swift

```diff
@@ -118,11 +116,50 @@ final class AutoResizingMaskViewSource {
     return rightFlexibleTopBottomView
   }()
 
-  private func bakeViewWithFrame(_ frame: CGRect) -> UIView {
-    let view = UIView(frame: frame)
+  private func bakeViewWithFrame(_ frame: CGRect) -> UserInterfaceStyleAwareView {
+    let view = UserInterfaceStyleAwareView(frame: frame)
     view.layer.cornerRadius = type(of: self).cornerRadius
     view.layer.borderWidth = type(of: self).borderWidth
-    view.layer.borderColor = type(of: self).borderColor
+    view.layer.borderColor = view.borderColor
     return view
   }
 }
+
+class UserInterfaceStyleAwareView: UIView {
```

Comment:
Would it make sense to make this `final`? If only used in this file I'd even argue whether it shouldn't be `private` instead?